test/unit/server/handlers/consents/{ID}/generateChallenge.test.ts

```diff
@@ -126,8 +127,8 @@ const completeConsentRevoked: Consent = {
 }
 
 const credential: ConsentCredential = {
-  credentialType: 'FIDO',
-  credentialStatus: 'PENDING',
+  credentialType: 'FIDO' as 'FIDO',
```

Comment:
Why this cast?

Reply:
We haven't declared a credentialTypeEnum - but I think after what we've done here - when we're updating type definitions we should refactor this there ?

Reply:
Yeah but I don't understand why you would need to cast a string to itself in the first place